test/test.js

```diff
@@ -44,6 +44,21 @@ const responses = {
   }
 }
 
+const cleanDOM = () => {
+  document.body.innerHTML = ''
+}
+
+const checkAsync = (done, func) => {
```

Comment:
This is only ever used in combination with `addEventListener`. Also, its invocation style is a bit complex, since it needs manual combining of `done` callback, `addEventListener` and `checkAsync` (whose name is kind of vague):

```js
    div.firstChild.addEventListener('load', function() {
      checkAsync(done, () => {
        // ...
        cleanDOM() // this looks like it's superfluous, btw?
      })
    })
```

How about a Promise-based approach?

```js
function when(el, eventType) {
  return new Promise(function(resolve) {
    el.addEventListener(eventType, resolve)
  })
}

// usage:
test('...', function() {
  return when(div.firstChild, 'load').then(() => {
    assert.equal(document.querySelector('include-fragment'), null)
  })
})
```